docs/api/rest_api/rest_api_authentication.md

```diff
@@ -18,6 +18,11 @@ For other security related subjects, see:
 - [Cross-origin requests](rest_responses.md#cross-origin)
 - [`access_control`]([[= symfony_doc =]]/security/access_control.html)
 
+!!! caution "SiteAccess login"
+
+    The anonymous user is used to perform authentification requests.
+    Therefore, the "Anonymous" role must have `user/login` permission on the SiteAccess that matches the REST domain or is passed through the [`X-Siteaccess` header](rest_requests.md#siteaccess).
+
```

Comment:
https://github.com/ibexa/documentation-developer/pull/2505#discussion_r1784593510

Reply:
A bit hard to test. It doesn't harm too much to have this remark for every methods even if some method may not need it.